src/main/java/uk/gov/digital/ho/hocs/search/application/RequestData.java

```diff
@@ -13,25 +13,13 @@
 @Component
 public class RequestData implements HandlerInterceptor {
 
-
     public static final String CORRELATION_ID_HEADER = "X-Correlation-Id";
     public static final String USER_ID_HEADER = "X-Auth-UserId";
     public static final String USERNAME_HEADER = "X-Auth-Username";
     public static final String GROUP_HEADER = "X-Auth-Groups";
 
     private static final String ANONYMOUS = "anonymous";
```

Comment:
Is `anonymous` actually useful in this context, I'd rather just pass through empty that some arbitrary value...